python/interpret_community/widget/js/main.js

```diff
@@ -34,6 +34,7 @@ const RenderDashboard = (divId, data) => {
       }}
       requestPredictions={data.predictionUrl !== undefined ? generatePrediction : undefined}
       locale={data.locale}
+      explanationMethod={data.explanation_method}
```

Comment:
The new prop we need to add to dashboardProps as a result of this PR is "explanationMethod" or "explanation_method"?

Reply:
What happens if this prop is omitted? Is there a default value within the dashboard?

Reply:
the key on the interface is explanationMethod. The serialized python object has the property explanation_method, that will have no impact on the studio side or any other user, just set the explanationMethod stirng when available.

Reply:
Looking at IExplanationDashboardProps, it looks like method is part of IModelInformation.

export interface IModelInformation {
    modelClass: 'Tree' | 'EBM' | 'blackbox';
    method: 'classifier' | 'regressor';
}

Is this where method goes?
Is modelClass also needed?

Reply:
that is about the model, as either a classifier or regressor. This new parameter is about the explanation type (LIME, SHAP, Mimic, all of which could be working on a classifier or regressor model). Walter said this is currently being uploaded, but it may not yet be on the vis_dict object @wamartin-aml ?